python/benchmark/dataproc/run_benchmark.sh

```diff
@@ -60,7 +60,8 @@ if [[ $? != 0 ]]; then
     exit 1
 fi
 
-cluster_name=${USER}-spark-rapids-ml-${cluster_type}
+cluster_name=${USER}-spark-rapids-ml-${cluster_type}-{GITHUB_RUN_NUMBER}
```

Comment:
nit: looks like this script is not for CI only. could try some other way like letting user provide the naming
```
cluster_name=${CLUSTER_NAME:-"${USER}-spark-rapids-ml-${cluster_type}"}
```